run.sh

```diff
@@ -164,7 +172,18 @@ az keyvault create --name ${VAULT_NAME} \
   --enabled-for-deployment true  \
   --tags "${COMMON_TAGS[@]}" \
   --enabled-for-template-deployment true
-  
+
+if [ ${SUB} == "cftapps" ]; then
```

Comment:
I don't think this if is needed you're missing cftsbox and cftptl with this

Reply:
The if statement has been removed